manager/manager/settings.py

```diff
@@ -88,14 +88,14 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': POSTGRES_DB,
-        'USER': POSTGRES_USER,
-        'PASSWORD': POSTGRES_PASSWORD,
-        'HOST': '127.0.0.1',
-        'PORT': '5432',
-        # 'ENGINE': 'django.db.backends.sqlite3',
-        # 'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        # 'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        # 'NAME': POSTGRES_DB,
+        # 'USER': POSTGRES_USER,
+        # 'PASSWORD': POSTGRES_PASSWORD,
+        # 'HOST': '127.0.0.1',
+        # 'PORT': '5432',
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
Are we not using Postgres?

Reply:
this is just for local. Heroku has its own way of doing databases. Basically it automatically uses the PSQL on the app